commands/getUser.js

```diff
@@ -54,34 +54,35 @@ const whoami = async function (userid, server, Discord) {
   } catch (e) {
     console.error(e.toString());
     await db.close();
-    return [{ message: e.toString() }, null];
+    return [{ message: e.toString() }, null, false];
   }
   await db.close();
   if (!row) {
     return [
       null,
-      `
-Hmmm I'm really not sure myself but I'd love to get to know you!
-Use \`!iam <affiliation> <name> <ucla_email>\` and verify your email address.`,
+      `Hmmm I'm really not sure myself but I'd love to get to know you!
```

Comment:
The new line between the backticks and text actually shows because backticks are for string "literals".

Reply:
this is an issue that exists in master as well. Took the opportunity to fix it rn